doc/release-notes/10517-datasetType.md

```diff
@@ -0,0 +1,14 @@
+### Initial Support for Dataset Types
+
+Out of the box, all datasets have the type "dataset" but superusers can add additional types. At this time the type can only be set at creation time via API. The types "dataset", "software", and "workflow" will be sent to DataCite when the dataset is published.
+
+For details see <https://dataverse-guide--10694.org.readthedocs.build/en/10694/user/dataset-management.html#dataset-types> and #10517. Please note that this feature is highly experimental and is expected to evolve.
+
+Upgrade instructions
+--------------------
+
+Add the following line to your Solr schema.xml file and do a full reindex:
```

Comment:
With the line already in the schema.xml above, this should probably evolve into the standard instructions to update schema.xml, or, with custom blocks, to run the update script.  Adding the one (or two) lines to the existing schema.xml is an option (assuming there are no other schema.xml changes n v6.4) but I don't think we usually call that out.

Reply:
I made the instructions more generic in 260ac3acd8.

Yes, the normal boilerplate should be used. This snippet is just a placeholder.